### DISCUSSION TITLE
How to set label position? It always confuse me for jump target poistion.

### DISCUSSION THREAD

**Discussion by jqhr** [ggandor/leap.nvim#140] [Q&A] [answered]
The label's position is at the next charactoer  of pattern.For example:
![图片](https://user-images.githubusercontent.com/23305067/233757923-28742c44-81dd-4044-86fd-2b2f64c112a4.png)
I want to jump to "s", but label position is "f" after "sy". How to set label "f" over "s"?

**Comment by ggandor** [comment, marked as answer]
It's not possible, the label's position is hardcoded (at least now). The label is after the 2nd char, since (according to most people, including me) in case of ahead-of-time labeling - when you see the label _while_ you're typing the pattern -, it's much more natural to see the three characters in the same order as you need to type them, instead of doing a microscopic horizontal U-movement with your eyes.

If you set `opts.max_phase_one_targets = 0`, that is, disable ahead-of-time labeling, the labels is displayed over the first character.
